CHANGELOG.md

```diff
@@ -2,43 +2,104 @@
 
 ## 0.3.0 (2018-03-30)
 
-### Runtime
-
-* Add plugin framework which supports executing custom code at plugin points
-* Add daemon update API to update pouch daemon's labels, image-proxy and so on
-* Add support set OOM options for container
+**IMPORTANT**: Pouch 0.3.0 has met almost all your basic needs for kubernetes:
+
+1. Sandbox/Container lifecycle management
+2. Image management
+3. Network management with CNI
+4. Container streaming: exec/attach/portforward
+5. Container logging
+6. Security Context: RunAsUser, Apparmor,Seccomp,Sysctl
+7. Validation test of cri-tools: 51/55 Pass
+8. Kubernetes node-e2e test: 82 Pass, 104 Failed, 0 Pending, 72 Skipped
+
+**IMPORTANT**:
+Kubernetes 1.10 was released recently and the CRI of it has updated from v1alpha1 to v1alpha2.
+Pouch will first try to fully support CRI v1alpha1 and then CRI v1alpha2.
```

Comment:
Pouch will first try to fully support CRI v1alpha1 and then CRI v1alpha2, so kubernetes 1.9.X is recommended now.

s/recommanded/recommended